app/lib/shared/configuration.dart

```diff
@@ -77,18 +84,21 @@ class Configuration {
   factory Configuration._prod() {
     final projectId = 'dartlang-pub';
     return Configuration(
-        projectId: projectId,
-        packageBucketName: 'pub-packages',
-        dartdocStorageBucketName: '$projectId--dartdoc-storage',
-        popularityDumpBucketName: '$projectId--popularity',
-        searchSnapshotBucketName: '$projectId--search-snapshot',
-        backupSnapshotBucketName: '$projectId--backup-snapshots',
-        searchServicePrefix: 'https://search-dot-$projectId.appspot.com',
-        pubHostedUrl: 'https://pub.dartlang.org',
-        pubClientAudience: _pubClientAudience,
-        pubSiteAudience:
-            '818368855108-e8skaopm5ih5nbb82vhh66k7ft5o7dn3.apps.googleusercontent.com',
-        credentials: _loadCredentials());
+      projectId: projectId,
+      packageBucketName: 'pub-packages',
+      dartdocStorageBucketName: '$projectId--dartdoc-storage',
+      popularityDumpBucketName: '$projectId--popularity',
+      searchSnapshotBucketName: '$projectId--search-snapshot',
+      backupSnapshotBucketName: '$projectId--backup-snapshots',
+      searchServicePrefix: 'https://search-dot-$projectId.appspot.com',
+      pubHostedUrl: 'https://pub.dartlang.org',
+      pubClientAudience: _pubClientAudience,
+      pubSiteAudience:
+          '818368855108-e8skaopm5ih5nbb82vhh66k7ft5o7dn3.apps.googleusercontent.com',
+      credentials: _loadCredentials(),
+      searchEnginesEnabled: true,
```

Comment:
Wouldn't be it better to rename this `blockRobots` and the serve a `robots.txt` that blocks robots from the staging site, I assume that is the purpose here?

Reply:
Also, I don't know maybe `searchEnginesEnabled` isn't the best name... when this service also features a search engine :)

Could we call it `robotsTxtEnabled`?

Reply:
Or something like that... the "Txt" part was particularly ugly, hehe :)

Reply:
`blockRobots` sounds better, renaming